charts/kubernetes-agent/README.md

```diff
@@ -1,7 +1,6 @@
 # kubernetes-agent
 
-
-![Version: 1.0.3](https://img.shields.io/badge/Version-1.0.3-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 8.1.1588](https://img.shields.io/badge/AppVersion-8.1.1588-informational?style=flat-square) 
+![Version: 1.0.4](https://img.shields.io/badge/Version-1.0.4-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 8.1.1588](https://img.shields.io/badge/AppVersion-8.1.1588-informational?style=flat-square)
```

Comment:
You may want to update to `main` because this should already by `1.0.4`

Reply:
Currently in `main` the readme still says `1.0.3` https://github.com/OctopusDeploy/helm-charts/blob/640dc033cd36dea75f88f0103206a1595a703c6b/charts/kubernetes-agent/README.md?plain=1#L4 

I think it's because the versioning bump happens in a separate PR to when the readme is generated e.g. https://github.com/OctopusDeploy/helm-charts/pull/147 didn't regenerate the readme after changing the version numbers

Reply:
This should be fixed with https://github.com/OctopusDeploy/helm-charts/pull/150 so going forward it should update the readme on versioning PRs